src/template-tags/tickets.php

```diff
@@ -568,3 +568,91 @@ function tribe_tickets_get_template_part( $slug, $name = null, array $data = nul
 	}
 }
 
+
+if ( ! function_exists( 'tribe_tickets_get_event_ids' ) ) {
+
+	/**
+	 * Gets an array of event ids when passing an id
+	 *
+	 * @param integer /string $id a rsvp order key, order id, attendee id, ticket id, or product id
+	 *
+	 * @return array
+	 */
+	function tribe_tickets_get_event_ids( $id ) {
+		return Tribe__Tickets__Data_API::get_instance()->get_event_ids( $id );
+	}
+}
+
+if ( ! function_exists( 'tribe_tickets_get_ticket_provider' ) ) {
+
+	/**
+	 * Gets the ticket provider class when passed an id
+	 *
+	 * @param integer /string $id a rsvp order key, order id, attendee id, ticket id, or product id
+	 *
+	 * @return bool/object
+	 */
+	function tribe_tickets_get_ticket_provider( $id ) {
+		return Tribe__Tickets__Data_API::get_instance()->get_ticket_provider( $id );
+	}
+}
+
+if ( ! function_exists( 'tribe_tickets_get_attendees' ) ) {
+
+	/**
+	 * Get attendee(s) by an id
+	 *
+	 * @param $id  integer/string $id a rsvp order key, order id, attendee id, ticket id, or event id
+	 * @param $context null use 'rsvp_order' to get all rsvp tickets from an order based off the post id and not the order key
```

Comment:
In this instance you would pass a post id and 'rsvp_order' and the coding would go look for a order hash and return all rsvp's for this order. This is a way to bypass the fact there is no true RSVP Order Post like with EDD and Woo and you can simply pass an id of an rsvp and state you are using it as a rsvp order and not a single rsvp post.